### PR TITLE
Update 128 bit logging enabled tests and disable stable config tests

### PR DESCRIPTION
**Pull request by mhlidd**
## Motivation

Update 128bit logging enabled tests to consider the Java edge case. Update and disable stable config Java tests to allow CI to pass on the following dd-trace-java [PR](https://github.com/DataDog/dd-trace-java/pull/8489) that updates the default value of `DD_LOGS_INJECTION`. Stable Config tests will be enabled in a future PR following the merge of the above dd-trace-java PR.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Review by mabdinur — APPROVED** [review]
Left some questions and minor suggestions. Feel free to merge the PR when CI is green.

File: manifests/java.yml [hunk 1]
```diff
@@ -1823,8 +1823,12 @@ tests/:
     Test_Config_IntegrationEnabled_True:
       '*': irrelevant (kafka endpoints are not implemented)
       spring-boot: v1.42.0
-    Test_Config_LogInjection_128Bit_TraceId_Default: missing_feature (disabled by default)
-    Test_Config_LogInjection_128Bit_TraceId_Disabled: incomplete_test_app (weblog endpoint not implemented)
+    Test_Config_LogInjection_128Bit_TraceId_Default:
+      '*': incomplete_test_app (weblog endpoint not implemented)
+      'spring-boot-3-native': missing_feature
```
[the remark this answers is not in the document]
Comment (mhlidd) [reply manifests/java.yml:1828]: Oh we could do it like this, but I was going open a new PR to enable all tests together anyways. This was just to update the manifests to help Java CI pass
